conf/modules/adc_expansion_uart.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="adc_expansion_uart">
+  <doc>
+    <description>Get analog signals from expansion board to main autopilot over uart</description>
```

Comment:
and some more details on how it works and how it's supposed to be used also wouldn't hurt ;-)
I.e. that you should send the ADC_DATA message via datalink and how this modules then uses that info...